client/src/components/LocalEmergencyBanner/localEmergencyData.js

```diff
@@ -0,0 +1,1456 @@
+    "Ambulance": 999,
+    "Fire": 999
+  },
+  {
+    "Country": "Sao Tome and Principe",
+    "Police": 112,
+    "Ambulance": 112,
+    "Fire": 112
+  },
+  {
+    "Country": "Senegal",
+    "Police": 17,
+    "Ambulance": 18,
+    "Fire": 1515
+  },
+  {
+    "Country": "Seychelles",
+    "Police": "112/999",
+    "Ambulance": "112/999",
+    "Fire": "112/999"
+  },
+  {
+    "Country": "Sierra Leone",
+    "Police": 19,
+    "Ambulance": 999,
+    "Fire": 999
+  },
+  {
+    "Country": "Somalia",
+    "Police": 888,
+    "Ambulance": 999,
+    "Fire": 555
+  },
+  {
+    "Country": "South Africa",
+    "Police": "10 111",
+    "Ambulance": "10 177",
+    "Fire": "10 177"
+  },
+  {
+    "Country": "Sudan",
+    "Police": 999,
+    "Ambulance": 999,
+    "Fire": 999
+  },
+  {
+    "Country": "South Sudan",
+    "Police": 999,
+    "Ambulance": 999,
+    "Fire": 999
+  },
+  {
+    "Country": "Swaziland",
+    "Police": 999,
+    "Ambulance": 977,
+    "Fire": 933
+  },
+  {
+    "Country": "Tanzania",
+    "Police": 112,
+    "Ambulance": 114,
+    "Fire": 115
+  },
+  {
+    "Country": "Togo",
+    "Police": 117,
+    "Ambulance": 8200,
+    "Fire": 118
+  },
+  {
+    "Country": "Tristan da Cunha",
+    "Police": 999,
+    "Ambulance": 911,
+    "Fire": 999
+  },
+  {
+    "Country": "Tunisia",
+    "Police": 197,
+    "Ambulance": 198,
+    "Fire": 190
+  },
+  {
+    "Country": "Uganda",
+    "Police": 112,
+    "Ambulance": 911,
+    "Fire": 112
+  },
+  {
+    "Country": "Western Sahara",
+    "Police": 150,
+    "Ambulance": 150,
+    "Fire": 150
+  },
+  {
+    "Country": "Zambia",
+    "Police": 999,
+    "Ambulance": 999,
+    "Fire": 999
+  },
+  {
+    "Country": "Zimbabwe",
+    "Police": 999,
+    "Ambulance": 999,
+    "Fire": 999
+  },
+  {
+    "Country": "Antarctica",
+    "Police": 911,
+    "Ambulance": 911,
+    "Fire": 911
+  },
+  {
+    "Country": "Abkhazia",
+    "Police": 102,
+    "Ambulance": 103,
+    "Fire": 101
+  },
+  {
+    "Country": "Afghanistan",
+    "Police": 119,
+    "Ambulance": 112,
+    "Fire": 119
+  },
+  {
+    "Country": "Akrotiri and Dhekelia",
+    "Police": "112/999",
+    "Ambulance": "112/999",
+    "Fire": "112/999"
+  },
+  {
+    "Country": "Bahrain",
+    "Police": 999,
+    "Ambulance": 999,
+    "Fire": 999
+  },
+  {
+    "Country": "Bangladesh",
+    "Police": 999,
+    "Ambulance": 999,
+    "Fire": 999
+  },
+  {
+    "Country": "Bhutan",
+    "Police": 113,
+    "Ambulance": 112,
+    "Fire": 110
+  },
+  {
+    "Country": "British Indian Ocean Territory",
+    "Police": "112/999",
+    "Ambulance": "112/999",
+    "Fire": "112/999"
+  },
+  {
+    "Country": "Brunei",
+    "Police": 993,
+    "Ambulance": 991,
+    "Fire": 995
+  },
+  {
+    "Country": "Myanmar",
+    "Police": 999,
+    "Ambulance": 999,
+    "Fire": 999
+  },
+  {
+    "Country": "Cambodia",
+    "Police": 117,
+    "Ambulance": 119,
+    "Fire": 118
+  },
+  {
+    "Country": "People's Republic of China",
```

Comment:
```suggestion
    "CN": {...
```

I think we should be using standard country codes instead - see for https://extreme-ip-lookup.com/json/1.185.255.255 - the country name is "China" by "CN" is definitely standard so should be better.

We can also make it an object instead of an array so each key is a unique country code.

Reply:
Hi @joshmorel awesome! Once it's all looked over I'll go ahead and make the changes and do another pull req

Reply:
Hi @joshmorel I've updated the code and have pushed it up. Hope all is good now!

Reply:
This seems like it's working well locally with once exception (I will comment elsewhere)... too bad the review branch isn't working.